apps/server/src/modules/task-card/controller/api-test/task-card.api.spec.ts

```diff
@@ -411,11 +363,71 @@ describe('Task-Card Controller (api)', () => {
 					.expect(200);
 
 				const responseTaskCard = response.body as TaskCardResponse;
-				const richTextElement = responseTaskCard.cardElements.filter(
+				const richTextElement = responseTaskCard.cardElements?.filter(
 					(element) => element.cardElementType === CardElementType.RichText
 				);
-				expect((richTextElement[0].content as CardRichTextElementResponse).value).toEqual(sanitizedText);
+				expect(richTextElement ? richTextElement[0].content.value : '').toEqual(sanitizedText);
 			});
 		});
 	});
+	describe('When title is provided', () => {
+		it('should create a task card with title', async () => {
+			const user = setupUser([Permission.TASK_CARD_EDIT, Permission.HOMEWORK_CREATE, Permission.HOMEWORK_EDIT]);
+
+			await em.persistAndFlush([user]);
+			em.clear();
+
+			currentUser = mapUserToCurrentUser(user);
+
+			const taskCardParams = {
+				title: 'test title',
+			};
+
+			const response = await request(app.getHttpServer())
+				.post(`/cards/task/`)
+				.set('Accept', 'application/json')
+				.send(taskCardParams)
+				.expect(201);
+
+			const responseTaskCard = response.body as TaskCardResponse;
+
+			expect(responseTaskCard.title).toEqual(taskCardParams.title);
+		});
+		it('should throw an error if title is to short', async () => {
+			const user = setupUser([Permission.TASK_CARD_EDIT, Permission.HOMEWORK_CREATE, Permission.HOMEWORK_EDIT]);
+
+			await em.persistAndFlush([user]);
+			em.clear();
+
+			currentUser = mapUserToCurrentUser(user);
+
+			const taskCardParams = {
+				title: 't',
+			};
+
+			await request(app.getHttpServer())
+				.post(`/cards/task/`)
+				.set('Accept', 'application/json')
+				.send(taskCardParams)
+				.expect(400);
+		});
+		// it('should throw an error if title is not a string', async () => {
```

Comment:
next time please use block commenting so its not needed to comment out every single line